Makefile

```diff
@@ -1,22 +1,55 @@
-# Current Operator version
+# VERSION defines the project version for the bundle.
+# Update this value when you upgrade the version of your project.
+# To re-generate a bundle for another specific version without changing the standard setup, you can:
+# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
+# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
 VERSION ?= 0.0.1
-# Default bundle image tag
-IMAGE_TAG_BASE ?= <registry>/<operator name>
-BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:$(VERSION)
-# Options for 'bundle-build'
+
+# CHANNELS define the bundle channels used in the bundle.
+# Add a new line here if you would like to change its default config. (E.g CHANNELS = "candidate,fast,stable")
+# To re-generate a bundle for other specific channels without changing the standard setup, you can:
+# - use the CHANNELS as arg of the bundle target (e.g make bundle CHANNELS=candidate,fast,stable)
+# - use environment variables to overwrite this value (e.g export CHANNELS="candidate,fast,stable")
 ifneq ($(origin CHANNELS), undefined)
 BUNDLE_CHANNELS := --channels=$(CHANNELS)
 endif
+
+# DEFAULT_CHANNEL defines the default channel used in the bundle.
+# Add a new line here if you would like to change its default config. (E.g DEFAULT_CHANNEL = "stable")
+# To re-generate a bundle for any other default channel without changing the default setup, you can:
+# - use the DEFAULT_CHANNEL as arg of the bundle target (e.g make bundle DEFAULT_CHANNEL=stable)
+# - use environment variables to overwrite this value (e.g export DEFAULT_CHANNEL="stable")
 ifneq ($(origin DEFAULT_CHANNEL), undefined)
 BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
 endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
-# CI tools repo for running tests
-CI_TOOLS_REPO := https://github.com/openstack-k8s-operators/openstack-k8s-operators-ci
+# IMAGE_TAG_BASE defines the docker.io namespace and part of the image name for remote images.
+# This variable is used to construct full image tags for bundle and catalog images.
+#
+# For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
+# my.domain/foo-bundle:$VERSION and my.domain/foo-catalog:$VERSION.
+IMAGE_TAG_BASE ?= my.domain/foo
```

Comment:
Since this isn't commented-out, should we make this something specific to our project?

Reply:
Yeah, I was torn on setting this. It gets used to construct the BUNDLE_IMG and CATALOG_IMG which we typically explicitly set anyway in our scripts so it isn't required.

I guess we could set it to IMAGE_TAG_BASE ?= quay.io/osp-director-operator/osp-director-operator but I thought someone might inadvertently build and push to our default registry by mistake. WDYT?

Could add a comment to this regard though @abays